TelepresenceBot/mobile/src/main/java/com/novoda/tpbot/bot/UsbChangesListener.java

```diff
@@ -1,6 +1,6 @@
 package com.novoda.tpbot.bot;
 
-interface MovementService {
+interface UsbChangesListener {
```

Comment:
This had nothing to do with `MovementService`, it's the USB state. I guess technically this is unlikely to change and could just be a concrete class but wanted to expose all public interfaces using `interface`.